package.json

```diff
@@ -35,20 +35,21 @@
     "ember-cli-babel": "^6.10.0"
   },
   "devDependencies": {
-    "ember-cli": "^2.17.1",
     "ember-cli-dependency-checker": "^2.0.0",
     "ember-cli-eslint": "^4.2.1",
-    "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
+    "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-pretender": "^1.0.1",
-    "ember-cli-test-loader": "^2.2.0",
     "ember-cli-shims": "^1.2.0",
+    "ember-cli-test-loader": "^2.2.0",
+    "ember-cli": "^2.17.1",
     "ember-data": "~2.17.0",
     "ember-debug-handlers-polyfill": "^1.0.3",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-fetch": "^3.4.0",
     "ember-load-initializers": "^1.0.0",
+    "ember-maybe-import-regenerator": "^0.1.6",
```

Comment:
we should probably use https://github.com/ember-cli/ember-maybe-import-regenerator-for-testing instead since the production code should not use async/await

Reply:
Agreed, and just to be clear we do have linting setup to catch any `async` / `await` usage in `addon-test-support`...